server/fileserver.go

```diff
@@ -69,18 +57,19 @@ func getUploadHandler(fsPath string) http.Handler {
 					status = http.StatusInternalServerError
 					return
 				}
-			}()
-			_, err = io.Copy(out, req.Body)
-			if err != nil {
-				logger.WithError(err).Error("Error copying the request body to file (raw data from body)")
-				status = http.StatusInternalServerError
+				}()
+				_, err = io.Copy(out, req.Body)
+				if err != nil {
+					logger.WithError(err).Error("Error copying the request body to file (raw data from body)")
+					status = http.StatusInternalServerError
+					return
+				}
+				//write response
+				logger.Debug(fmt.Printf("writing the body from request %s to file", req.URL.Path))
+				writeResponse(res, req, http.StatusAccepted, &Response{fmt.Sprintf("Uploaded Files: %v. Total Bytes Written: %v", uploadedFileNames, totalBytesWritten)})
```

Comment:
don't pass uploadedFileNames to the response message formatting as it is empty in this case. use the name of the file you saved the body to